nbviewer/providers/base.py

```diff
@@ -607,7 +611,8 @@ def filter_formats(self, nb, raw):
     def finish_notebook(self, json_notebook, download_url, provider_url=None,
                         provider_icon=None, provider_label=None, msg=None,
                         breadcrumbs=None, public=False, format=None, request=None,
-                        title=None):
+                        title=None, executor_url=None, executor_label=None,
```

Comment:
Perhaps add to docstring that `title` is the title of the application that is displayed i.e. nbvviewer, and that it's not the metadata name of a notebook.

Reply:
`title` was already declared, but I'm all for tidying up a bit. I can doc all the parameters to that key method while I'm in there.

Reply:
Yeah, there was another comment elsewhere in the code that was interchangeably using title and notebook name.